src/main.py

```diff
@@ -52,13 +61,22 @@ def printCommandOptions() -> None:
     printGameMoves = 'gm: moves of current game in PGN format'
     quitOption = 'quit : resign'
     moveOption = 'a3, Nc3, Qxa2, etc : make the move'
-    options = [undoOption, printLegalMovesOption, randomMoveOption, printGameMoves,
-               quitOption, moveOption, '', ]
+    options = [
+        undoOption,
+        printLegalMovesOption,
+        randomMoveOption,
+        printGameMoves,
+        quitOption,
+        moveOption,
+        '',
+    ]
     print('\n'.join(options))
 
 
 def printAllLegalMoves(board: Board, parser: InputParser) -> None:
-    for move in parser.getLegalMovesWithNotation(board.currentSide, short=True):
+    for move in parser.getLegalMovesWithNotation(
+        board.currentSide, short=True
+    ):
```

Comment:
I guess this is debatable but I don't have a better option to offer, so it's ok.